### ISSUE TITLE
Notification Time

### ISSUE THREAD

**Issue by saltsoftdrink** [criticalAY/Neer#5]
Add a custom interval after which the user can drink water it should be customisable

**Comment by MonuGit9** [comment]
Can I work on it

**Comment by saltsoftdrink** [comment]
> Can I work on it

Yes, you can create a PR this will help the developer

**Comment by criticalAY** [comment]
It should be interval based, user can change the interval, project already has that interval option but I did not expose it via UI layer need to be handled, @saltsoftdrink if you want to do it then see the notification receiver, it would need to be updated in there along with pref manager,
off topic: This was started as my personal project but now I am planning to scale it so that others can use it too, there is no open source water reminder application ig

**Comment by saltsoftdrink** [comment]
> It should be interval based, user can change the interval, project already has that interval option but I did not expose it via UI layer need to be handled, @saltsoftdrink if you want to do it then see the notification receiver, it would need to be updated in there along with pref manager, off topic: This was started as my personal project but now I am planning to scale it so that others can use it too, there is no open source water reminder application ig

Yes, you should take this project seriously honestly people will be thankful for this also there is no FOSS app for this, you can take inspiration from Play store's app

I found 1 thing in Play Stores app they auto calculate how much the person will drink water instead of filling a random number it shows after entering Gender, height, weight

Thank You so much for creating this app ❤️😀

**Comment by saltsoftdrink** [comment]
> It should be interval based, user can change the interval, project already has that interval option but I did not expose it via UI layer need to be handled, @saltsoftdrink if you want to do it then see the notification receiver, it would need to be updated in there along with pref manager, off topic: This was started as my personal project but now I am planning to scale it so that others can use it too, there is no open source water reminder application ig

ig you should add that interval to UI also it will handy to customise

**Comment by shreyp135** [comment]
We can also add a custom notification sound so that just by hearing the notification the person can drink water without looking the phone.

**Comment by criticalAY** [comment]
Sounds resonable, but @saltsoftdrink  fyi you can update the app from Play Store and have this option now. Plus there are some UI updates too
